build

```diff
@@ -69,6 +70,32 @@ replace(path.join(__dirname, '.server-dist'), [
 replace(path.join(__dirname, './tools/set-import'), [
 	{regex: new RegExp(`(require\\\(.*?)(lib|sim)(.*?\\\))`, 'g'), replace: `$1.$2-dist$3`},
 ]);
+replace(path.join(__dirname, './tools/database-converter'), [
+	{regex: new RegExp(`(require\\\(.*?)(lib)(.*?\\\))`, 'g'), replace: `$1.lib-dist$3`},
+]);
+
+var databaseExisted = true;
```

Comment:
Does this logic and below belong in `build` and not `server/index`?

Reply:
I think so, yes. I feel like migrating and running conversion is part of the 'build' process and not so much 'starting' up the server process.

Reply:
There are multiple instances where we run build but have never start a server. see `pokemon-showdown` for example. Why do I need to migrate a database to `validate-team`?

Reply:
I think that's a valid part of the build. In practice, database migrations should be rare so it shouldn't come up much.